pkg/controllers/netpol/policy.go

```diff
@@ -476,6 +476,14 @@ func (npc *NetworkPolicyController) appendRuleToPolicyChain(policyChainName, com
 	args = append(args, "-m", "mark", "--mark", "0x10000/0x10000", "-j", "RETURN", "\n")
 	npc.filterTableRules[ipFamily].WriteString(strings.Join(args, " "))
 
+	// Extra logs to get more information about the policy dropping the packet via ulog2
+	logRuleComment := "\"rule to log dropped traffic\""
+	logComment := "\"DROP by policy: " + policyName + " in namespace: " + policyNamespace + "\""
```

Comment:
How about just namespace/name?
```suggestion
	logComment := "\"DROP by policy " + policyNamespace + "/" + policyName + "\""
```

Reply:
fantastic idea because we are limited by 64 characters :)

Reply:
Oh, that might be interesting since both name and NS can be up to 63 characters :/ Might have to use the UID or a substring of the UID instead? I think that's what's done for the kube-proxy chain names to keep them short. It'd be an extra step to find the policy but I'm not sure what else to do.

Reply:
Good point. I'll add a quick check and if the policyName or namespace is too long, then add a substring

Reply:
done!